pb/config/config.proto

```diff
@@ -579,7 +579,10 @@ message DashboardTab {
   DashboardTabFlakinessAlertOptions flakiness_alert_options = 24;
 
   // Configuration options for customizing dashboard tab status calculation.
-  // Used to define acceptable flakiness
+  // Options here define:
```

Comment:
Think you can leave this off, and let the underlying message define the options.